src/views/MetocDataView.vue

```diff
@@ -742,7 +754,7 @@ export default class MetocDataView extends Mixins(WMSMixin, TimeSeriesMixin) {
   }
 
   get showMap(): boolean {
-    const isMobileGraphOpen = this.hasSelectedLocation && this.$vuetify.breakpoint.mobile
+    const isMobileGraphOpen = this.hasDataToDisplay && this.$vuetify.breakpoint.mobile
```

Comment:
`hasDataToDisplay` was added to generalize this check for any data that has to be displayed, as one of the bugs fixed with this is data the map did not close on mobile breakpoints when a coordinate was selected.